WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -485,6 +508,19 @@ class MeViewController: UITableViewController {
     }()
 }
 
+// MARK: - WPSplitViewControllerDetailProvider Conformance
+
+extension MeViewController: WPSplitViewControllerDetailProvider {
+    func initialDetailViewControllerForSplitView(_ splitView: WPSplitViewController) -> UIViewController? {
+        // If we're not logged in yet, return app settings
+        guard let _ = defaultAccount() else {
+            return AppSettingsViewController()
```

Comment:
Is it is valid scenario? Is there a way to reproduce it?

Reply:
I don't think so, good catch. Will remove this